nextjs-ecommerce-oneclick/pages/api/cancelBuy.js

```diff
@@ -7,8 +7,7 @@ export default async function cancelBuy(req, res) {
     return;
   }
 
-  const connection = new Connection();
-  const client = new WorkflowClient(connection.service);
+  const client = new WorkflowClient();
   const workflow = client.getHandle(id);
```

Comment:
this one was also somewhat on purpose - any reasonable production deploy is going to want to specify the Connection, so was giving it as a nice starting point

Reply:
its neither here nor there so not too bothered

Reply:
I left it in hello world so people can learn about it, and the samples that had client options, but preferred it shorter when possible to help the SDK seem simple/easy. Figure early impressions more impt than easier to deploy (by then we've already hooked them 😄)